src/shogun/CMakeLists.txt

```diff
@@ -286,6 +286,20 @@ ELSE()
     SHOGUN_INCLUDE_DIRS(SCOPE PUBLIC SYSTEM ${rxcpp_INCLUDE_DIR})
 ENDIF()
 
+# TFLogger package
+FIND_PACKAGE(TFLogger 0.1.0 CONFIG)
+IF (TFLogger_FOUND)
+    SHOGUN_INCLUDE_DIRS(SCOPE PUBLIC SYSTEM ${TFLogger_INCLUDE_DIR})
```

Comment:
do we need really public system? as this is really gonna be just statically linked into .so... so afaik PRIVATE should be enough, no?